cairo/tests/utils/helpers.cairo

```diff
@@ -16,7 +16,9 @@ from src.utils.maths import unsigned_div_rem
 from src.utils.utils import Helpers
 
 namespace TestHelpers {
-    func init_evm_at_address{pedersen_ptr: HashBuiltin*, range_check_ptr}(
+    func init_evm_at_address{
+        pedersen_ptr: HashBuiltin*, range_check_ptr, initial_state: model.State*
```

Comment:
why in implicit arg ?

Reply:
No need  refactored with [ff29eac](https://github.com/kkrt-labs/keth/pull/249/commits/ff29eac5a329d926d39f8c0c000d32cab415d046)